src/components/Card/Card.tsx

```diff
@@ -6,25 +6,38 @@ interface CardProps {
   image: string
   id: number
   sentiment: number
+  topic: string
   updateUserSentiment: (userSentiment: number) => void
 }
 
-const Card = ({ title, image, id, sentiment, updateUserSentiment }: CardProps): JSX.Element => {
+const Card = ({ title, image, id, sentiment, topic, updateUserSentiment }: CardProps): JSX.Element => {
+  console.log(sentiment);
   return (
-
+    <div className="article-boundary">
       <div className="card-container">
         <article className="news-card cy-article-card">
           <Link
             to={`/feed/${id}`}
             onClick={() => updateUserSentiment(sentiment) }
             className="cy-article-link"
           >
+          <div className="article-image-container">
```

Comment:
I really like how the sentiment score looks on each card. Good job!

Reply:
I think needs to be added to the TopicCard component too. Only the home/top news section displays the sentiment score.

Reply:
Cool, I'll look into that. :)